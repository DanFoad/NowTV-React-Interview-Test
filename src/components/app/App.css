/**** APP ****/
.main {
  display: flex;
  flex-flow: row nowrap;
  align-items: stretch;
  height: 100%;
  width: 100%;
  overflow: hidden;
}
