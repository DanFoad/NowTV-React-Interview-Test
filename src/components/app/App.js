import React, { Component } from 'react';

import MessagesContainer from '../../containers/messages/MessagesContainer';

import './App.css';

export default class App extends Component {
  render() {
    return (
      <div className="main">
        <MessagesContainer />
      </div>
    );
  }
}
