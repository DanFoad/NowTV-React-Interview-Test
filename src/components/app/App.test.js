import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import App from './App';

const renderer = new ShallowRenderer()

it('renders correctly', () => {
  const tree = renderer
    .render(<App />)
  expect(tree).toMatchSnapshot()
});