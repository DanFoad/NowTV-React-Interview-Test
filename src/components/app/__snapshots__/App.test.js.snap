exports[`test renders correctly 1`] = `
<div
  className="main">
  <Connect(MessagesContainer) />
</div>
`;
